### DISCUSSION TITLE
Grouping

### DISCUSSION THREAD

**Discussion by BeOSR** [mateusz-bak/openreads#737] [Ideas]
**Is your feature request related to a problem? Please describe.**
The standard list and grid views of books can be rather unorganised, and even though they can be filtered, they cannot be grouped.  

**Describe the solution you'd like**
It would be great to be able to group the books, e.g. per year, quarter, month, week or day (this last one is rather unlikely but added for completeness). Expanding the view or not would only show the period you're interested in viewing, without having to scroll through the entire collection.

**Additional context**
The app Book catalogue groups by year as standard, and this provides much more convenient and clear way of presenting books, especially bigger collections.

**Comment by jwildman16** [comment]
I'd like to see grouping by author.

**Comment by sjiawjbssj** [comment]
I'd love to group by series. That would require a new field when adding books though.
